deploy/operator/base/rbac.yaml

```diff
@@ -107,10 +107,20 @@ rules:
       - patch
       - list
       - watch
+#  - apiGroups:
```

Comment:
minor nit, but IMHO we should just delete stuff like this instead of commenting it out, it's all still in the history anyway.

Reply:
removed, along with some commented out yaml in the config yaml files